tokio-trace/tokio-trace-core/src/subscriber.rs

```diff
@@ -187,6 +208,12 @@ pub trait Subscriber: 'static {
 
     /// Records that an [`Event`] has occurred.
     ///
+    /// This method will be invoked, when a `trace!`, `info!` or other event is constructed.
+    ///
+    /// The key difference between this method and `record` is that an event refers to the specific
```

Comment:
I would have just said here that the difference between this method and `record` is that `record` is called when a value is recorded for a field defined by a span, while `event` is called when a new event occurs. 

I think the semantic difference between recording a field on a span and recording an event ought to be discussed in the documentation for the `Event` type, not here. The docs on `Subscriber` could link to that.

Reply:
This makes sense!

`Event` is already linked above, so i didn't include it as well